inlong-audit/conf/audit-proxy-pulsar.conf

```diff
@@ -61,6 +61,8 @@ agent1.sinks.pulsar-sink-msg1.channel = ch-msg1
 agent1.sinks.pulsar-sink-msg1.type = org.apache.inlong.audit.sink.PulsarSink
 agent1.sinks.pulsar-sink-msg1.pulsar_server_url = pulsar://localhost:6650
 agent1.sinks.pulsar-sink-msg1.topic = persistent://public/default/inlong-audit
+agent1.sinks.pulsar-sink-msg1.enable_auth = true
+agent1.sinks.pulsar-sink-msg1.auth_token = token
```

Comment:
the default value change to an empty value.

Reply:
enable_auth
->
enable_token_auth

Reply:
fix